session/server.go

```diff
@@ -125,3 +129,20 @@ func (s *Server) Stop() {
 		delete(s.activeSessions, id)
 	}
 }
+
+func (s *Server) SubscribeToSessionClosed(
+	localPublicKey *btcec.PublicKey) chan struct{} {
+
+	s.activeSessionsMtx.Lock()
+	defer s.activeSessionsMtx.Unlock()
+
+	var id sessionID
+	copy(id[:], localPublicKey.SerializeCompressed())
+
+	session, ok := s.activeSessions[id]
+	if !ok {
+		return nil
```

Comment:
To avoid this nil channel from being returned, should we instead return `chan struct{}` in `StartSession`?